### ISSUE TITLE
Fonts for the answer should be the same to question!

### ISSUE THREAD

**Issue by adrianq** [EyeSeeTea/SurveillanceCambodiaApp#59]
Feedback from qispsk@gmail.com : Fonts for the answer should be the same to question!

<img src="https://s3-us-west-2.amazonaws.com/buddybuild-screenshots/56d792f0e4d6010100721090/57586bc3bf09ff010013bc3a/5a2dab74-e943-4ec0-855e-a06e43744b6e.jpg" width="33%" height="33%" /><table><tr><td>Build</td><td>25</td></tr><tr><td>Device type</td><td>LG-H324</td></tr><tr><td>Screen size</td><td>480</td></tr><tr><td>Screen size</td><td>480px by 786px</td></tr><tr><td>Battery</td><td>54% Unplugged</td></tr><tr><td>Memory free</td><td>137 MB / 711 MB</td></tr></table>

 [Link to buddybuild feedback from build 25](https://dashboard.buddybuild.com/apps/56d792f0e4d6010100721090/feedback?fid=57591e0e32322f010011ba15&bnum=25)

**Comment by ifoche** [comment]
@QISPSK the options have the text embedded in the images you provided to us. Maybe you want to provide us new images using the KhmerOS.ttf font that is the one used for the rest of the application?

**Comment by rodmelia** [comment]
As ignacio is pointing out, the font in the app has not been changed, and we don't want to change it now. If you want a new font, you should provide the file compatible with Android, and we can do the change for the next version.
We recommend that you update your graphics with a font visually compatible with the one in the app. Changing graphics is easy and low risk.

**Comment by rodmelia** [comment]
@adrianq  will send the KhmerOS.ttf font so the art work uses the same font as the application

**Comment by QISPSK** [comment]
Unfortunately the KhmerOS.tff font does not seem to allow for sub-consonants in Adobe Illustrator. I've moved this issue to S6 as it looks like we will need to change the font of the questions (and the answers if the Khmer.tff file we used for the images is not compatible with Android) in the next release of the app.

**Comment by QISPSK** [comment]
![image](https://cloud.githubusercontent.com/assets/17549893/25211225/6096aa7e-25ae-11e7-8fa0-c985c75fffb9.png)

Moved back to "Backlog" as this remains an issue.

Given that these are images, this may not be possible to change, however, would it be possible to convert these to text in a later update (say, S7) that would allow consistent font sizes or would this create issues with scaling to different screen sizes/resolutions?

**Comment by ifoche** [comment]
@QISPSK yes, we could use floating text on top of the options and that text would adapt to font sizes. The solution we've experimented for that in Laos MCS App, had the only inconvenient that sometimes text overlapped the image, so you need to choose a good color contrast between image and text if you want to directly use that solution (just in case you open the app from a device with a small screen). Otherwise, we could also develop a different solution based on writing text always in a fix position relative to the image instead of floating over the image. Of course, the second option would imply more time than the first one. I assume then that we can change milestone to v1.4 instead of v1.3 for this issue, right?

**Comment by QISPSK** [comment]
@ifoche Yes, I believe we can move this to v1.4 instead as the current build text size works just fine, it just isn't consistent. Would having a fixed text box allow for the text to dynamically adjust similar to the question text size? 

If so, I think this is something we should explore and budget out for v1.4 as text size (particularly answer text size) has been a consistent headache for providers.

**Comment by rodmelia** [comment]
Yes - this is to be considered in next version- part of the challenge is
that location of text changes for landscape vs portrait, and color of font
also needs to change.

**Comment by ifoche** [comment]
@QISPSK yes, both options (floating text over the image and text in a fixed position relative to the image) would allow the text to adjust similar to the question text size. I've moved this to v1.4 following what you and @rodmelia said in your comments
